src/axom/core/docs/sphinx/core_acceleration.rst

```diff
@@ -0,0 +1,51 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core acceleration
+******************************************************
+
+Axom Core provides support to ease the implementation of user applications that
+intend to support execution utilizing accelerators. In order to supply this,
+Axom provides a simple API to access memory and computation, using RAJA
+and Umpire to access accelerators.
+
+.. warning:: 
+   You do need to compile with RAJA and Umpire to fully leverage this Axom functionality. 
+   While both memory management and execution spaces in Axom will operate without these tools, they will offer little.
+
+The memory management submodule offers a singular interface which allows the user to leverage either C++ memory functions or Umpire, 

```

Comment:
```suggestion
The memory management API allows the user to leverage either C++ memory functions or Umpire, 
```

Reply:
Suggestion: Adding subsections for memory management and execution spaces might make it easier for readers to get a quick feel for the page.

Reply:
I lightly rephrased this part.

Reply:
I think this documentation can always be improved by further restructuring or editing, but will serve its purpose as-is.